pkg/oc/experimental/dockergc/dockergc.go

```diff
@@ -0,0 +1,265 @@
+package dockergc
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+
+	dockerapi "github.com/docker/engine-api/client"
+	dockertypes "github.com/docker/engine-api/types"
+	dockerfilters "github.com/docker/engine-api/types/filters"
```

Comment:
as long as these dependencies 1) already exist and 2) aren't ones we're trying to delete as part of the oc split (I don't think they are... @deads2k?), hanging this under experimental doesn't bother me

Reply:
> as long as these dependencies 1) already exist and 2) aren't ones we're trying to delete as part of the oc split (I don't think they are... @deads2k?), hanging this under experimental doesn't bother me

We do want to see this leave as part of `oc`.  Is this command logically an `oc adm` command? If so, then `openshift ex` is ok and we'll move it to `oc ex` later.

Reply:
@deads2k it should actually be `oc` but `openshift ex`.  the command is temporary and is avoiding us having to ship an official product image.

@sjenning i think /pkg/cmd/openshift experimental is the right home for this.  similar to those other experimental commands.

Reply:
moved discussion to irc.  agree w/ @deads2k that we may want to handle this similar to kubernetes-namespace-reservation

Reply:
@sjenning -- @deads2k and i spoke in person.  when `oc` moves out of tree, we will move this command with it.  at that point, we will need to make sure our ops team has the updated daemonset.